src/meshapi/templates/widgets/panorama_viewer.html

```diff
@@ -1,5 +1,15 @@
-<div style='display:flex; flex-direction: row; width:100%; overflow-x:auto; max-height:300px; padding-left: 10px;'>
-	{% for panorama in widget.value %}
-		<a href="{{ panorama }}" target="_blank"><img src="{{ panorama }}" style="margin-right:10px"/></a>
-	{% endfor %}
+<script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js"></script>
```

Comment:
Can this safely use SRI?

Reply:
I'm not sure. I could probably serve a copy of flickity from our server if that would be better.